test/Microsoft.Extensions.Configuration.EnvironmentVariables.Test/EnvironmentVariablesTest.cs

```diff
@@ -67,13 +67,13 @@ public void LoadKeyValuePairsFromAzureEnvironment()
             string value;
             Assert.Equal("TestAppName", envConfigSrc.Get("APPSETTING_AppName"));
             Assert.False(envConfigSrc.TryGet("AppName", out value));
-            Assert.Equal("CustomConnStr", envConfigSrc.Get("Data:db1:ConnectionString"));
-            Assert.Equal("SQLConnStr", envConfigSrc.Get("Data:db2:ConnectionString"));
-            Assert.Equal("System.Data.SqlClient", envConfigSrc.Get("Data:db2:ProviderName"));
-            Assert.Equal("MySQLConnStr", envConfigSrc.Get("Data:db3:ConnectionString"));
-            Assert.Equal("MySql.Data.MySqlClient", envConfigSrc.Get("Data:db3:ProviderName"));
-            Assert.Equal("SQLAzureConnStr", envConfigSrc.Get("Data:db4:ConnectionString"));
-            Assert.Equal("System.Data.SqlClient", envConfigSrc.Get("Data:db4:ProviderName"));
+            Assert.Equal("CustomConnStr", envConfigSrc.Get("db1"));
+            Assert.Equal("SQLConnStr", envConfigSrc.Get("db2"));
+            Assert.Equal("System.Data.SqlClient", envConfigSrc.Get("db2:ProviderName"));
```

Comment:
You might be right, the build passes locally, but I added a debugger.launch that wasn't hit in these tests... I'll take a deeper look at why these tests aren't running in local build at least